.github/workflows/bokehjs-ci.yml

```diff
@@ -103,3 +103,40 @@ jobs:
         with:
           name: bokehjs-report
           path: bokehjs-report
+
+  test_docker:
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_TAG: bokeh/bokeh-dev:branch-3.1
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # full history to get proper build version
+
+      - name: Start Docker container, build Bokeh and run tests.
+        env:
+          BOKEH_DOCKER_BUILD: 1
+          BOKEH_DOCKER_TEST: 1
+          BOKEH_DOCKER_INTERACTIVE: 0
+        run: |
+          scripts/docker/docker_run.sh $IMAGE_TAG
```

Comment:
What exactly does this test? If we are testing bokehjs, then this shouldn't be building bokeh and running unrelated tests.

Reply:
Currently it runs
```
pytest tests/codebase
cd bokehjs && node make test
```

It could be changed to something else :shrug:

Reply:
OK, I just got confused by the job name "name: Start Docker container, build Bokeh and run tests." (at line 118). I thought this was copied from `bokeh-ci.yaml` and running wrong tests.